electrum_nmc/electrum/gui/qt/main_window.py

```diff
@@ -3360,9 +3363,23 @@ def update_buy_names_preview(self):
         identifier = self.buy_names_new_name_lineedit.text().encode('ascii')
         identifier_formatted = format_name_identifier(identifier)
         self.buy_names_preview_label.setText(_("Name to register: ") + identifier_formatted)
-
+        
+        # Convert to hexadecimal string
+        hex_text = name_to_str(identifier, Encoding.HEX)
```

Comment:
`hex_text` is kind of vague here; I think I'd prefer `identifier_hex` as the name of this variable.